pkg/volume/flexvolume/driver-call.go

```diff
@@ -235,3 +241,29 @@ func handleCmdResponse(cmd string, output []byte) (*DriverStatus, error) {
 
 	return &status, nil
 }
+
+// getDriverCapabilities returns the reported capabilities as returned by driver's init() function
+func getDriverCapabilities(plugin *flexVolumePlugin) (*driverCapabilities, error) {
+	call := plugin.NewDriverCall(initCmd)
```

Comment:
I'm fine with splitting out capabilities and initialization however it makes sense.

Storage vendors are beginning to depend on Flex as the sole way to interface with Kubernetes, and we don't have sufficient testing for the Flex interface yet. So my biggest concern is regressions: let's make sure that whatever we do, we do not break existing Flex drivers--not with this PR, not with future PRs.